fenix/app/src/androidTest/java/org/mozilla/fenix/ui/CreditCardAutofillTest.kt

```diff
@@ -0,0 +1,263 @@
+package org.mozilla.fenix.ui
+
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.fenix.customannotations.SmokeTest
+import org.mozilla.fenix.helpers.AndroidAssetDispatcher
+import org.mozilla.fenix.helpers.HomeActivityIntentTestRule
+import org.mozilla.fenix.helpers.TestAssetHelper
+import org.mozilla.fenix.helpers.TestHelper.exitMenu
+import org.mozilla.fenix.ui.robots.homeScreen
+import org.mozilla.fenix.ui.robots.navigationToolbar
+import java.time.LocalDate
+
+class CreditCardAutofillTest {
+    private lateinit var mockWebServer: MockWebServer
+
+    object FirstCreditCard {
+        const val MOCK_CREDIT_CARD_NUMBER = "5555555555554444"
+        const val MOCK_LAST_CARD_DIGITS = "4444"
+        const val MOCK_NAME_ON_CARD = "Mastercard"
+        const val MOCK_EXPIRATION_MONTH = "February"
+        val MOCK_EXPIRATION_YEAR = (LocalDate.now().year + 1).toString()
+    }
+
+    object SecondCreditCard {
+        const val MOCK_CREDIT_CARD_NUMBER = "2720994326581252"
+        const val MOCK_LAST_CARD_DIGITS = "1252"
+        const val MOCK_NAME_ON_CARD = "Mastercard"
+        const val MOCK_EXPIRATION_MONTH = "March"
+        val MOCK_EXPIRATION_YEAR = (LocalDate.now().year + 2).toString()
```

Comment:
Mind using similar mock data in https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/components/feature/prompts/src/test/java/mozilla/components/feature/prompts/creditcard/CreditCardsAdapterTest.kt ? I just want to be careful with credit card mock data.

Reply:
I left the credentials as they were. (I couldn't save the credit card with the credentials from the above link)

When the feature was **first** tested, my fellow colleagues added [pairing of credentials](https://testrail.stage.mozaws.net/index.php?/suites/view/3192&group_by=cases:section_id&group_id=227739&group_order=asc) that we can use.

![image](https://user-images.githubusercontent.com/51314259/220338177-929fc735-4eed-4a66-a21f-264bbd370b11.png)


Considering the fact that these are also used when performing manual tests, I would say it's safe to use them.

When I created the first "MockCreditCard" a while ago, I've used the same approach. ( used one of the pairings from our manual tests )